src/mumble/Log.cpp

```diff
@@ -531,15 +531,8 @@ QString Log::imageToImg(QImage img) {
 	QByteArray format = "PNG";
```

Comment:
If I see this correctly, the value `"PNG"` set here is never used. The variable's content seems to be overwritten with `JPEG"` before it is first used

Reply:
Yeah, you are totally right.